__tests__/app.test.js

```diff
@@ -166,7 +166,45 @@ describe("GET", () =>{
                         ]);
                     });
                 });
+
+                // test("responds with an empty array when given a valid ID that has no associated comments", () => {
+                //     return request(app)
+                //     .get("/api/articles/2/comments")
+                //     .expect(200)
```

Comment:
to get this test passing we need to do a separate check that the article id exists. If it doesnt we get a 404 if it does exist we get a 200 :)